docs/user-guide/security-features.md

```diff
@@ -1,152 +1,152 @@
+To block critical vulnerabilities globally but allow them in specific clusters:
 
-## Configure Global Policy
+1. Select the desired cluster.
+2. Change the critical setting to allow.
+3. This change only affects the policy of the selected cluster without impacting others or the global policy.
 
-To configure these policies, click on the drop-down option of the severity levels and select Block or Allow.
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/cluster-security-policy.jpg)
 
-![](../.gitbook/assets/security-feature8%20%282%29.png)
+## Configure Environment Security Policy
 
-## Configure Cluster Security Policy
+Environment Security Policies, like Cluster Security Policies, offer three options: Block, Allow, and Inherit.
 
-In the Global Security Policies, there are only two options available- Block and Allow. But in other options, we have an extra option named `Inherit`.
+The Environment Security Policy inherits its settings from the Cluster Security Policy, following a hierarchical structure where each level inherits the policy from its upper level.
 
-As the name itself gives you an idea about this option, it fetches the policy of its upper-level options, if we choose to inherit in the drop-down.
+When you select an environment, it automatically adopts the policy of the associated Cluster. For example, if critical-level vulnerabilities are blocked globally but allowed in the Cluster Security Policy, the Environment Security Policy will inherit this allowance. Consequently, critical-level vulnerabilities will also be allowed in the Environment Security Policy.
 
-Example-if you block critical severity levels in Global, then critical levels will be blocked in Cluster Security Policy. In case we change critical policy globally and allow it there, then it will be allowed in Cluster Security Policies too. But you can change these policies explicitly.
+However, you have the flexibility to make explicit changes to the policy if needed. This empowers you to customize the policy to align with specific requirements or preferences.
 
-If you want to block Critical Vulnerabilities in Global Security Policies but want to allow them in some clusters, then select your cluster and change the critical drop-down to allow. It will not affect the policy of other clusters and global also.
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/environment-security-policy.jpg)
 
-## Configure Environment Security Policy
+## Configure Application Security Policy
 
-Again we have three options to define a policy- Block, Allow, and Inherit.
+The Application Security Policy operates on a similar principle as other policies. However, in the Application Security Policy, the policy is determined by both the Environment option and the Application option.
```

Comment:
The `Application Security Policy` operates on a similar principle as other policies. However, in the `Application Security Policy`, the policy is determined by both the Environment option and the Application option.